compiler/rustc_interface/src/passes.rs

```diff
@@ -538,7 +538,7 @@ fn write_out_deps(tcx: TyCtxt<'_>, outputs: &OutputFilenames, out_filenames: &[P
             for path in out_filenames {
                 write!(
                     file,
-                    "{}: {}\n",
+                    "{}: {}",
```

Comment:
Hm. Why is this newline being removed?

Reply:
This was a `writeln!` with an additional newline, so it would be two newlines... I suppose that's why two are added later?

Reply:
It's put back down at line 568, it just needs to be not right there because the remainder of the body is filling out one single line.